scripts/generate_tree-sitter_bindings.py

```diff
@@ -0,0 +1,280 @@
+# TODO: port this to a Darklang script
```

Comment:
Could you document where this came from? I'm thinking also it doesn't belong in scripts, probably scripts/build?

Reply:
it's really not useful to us now; I'm going to delete it and bring a Dark version of it back later if useful